content/zh/docs/reference/access-authn-authz/rbac.md

```diff
@@ -0,0 +1,2137 @@
+and updates default cluster role bindings with any missing subjects.
+This allows the cluster to repair accidental modifications,
+and to keep roles and rolebindings up-to-date as permissions and subjects change in new releases.
+
+To opt out of this reconciliation, set the `rbac.authorization.kubernetes.io/autoupdate` 
+annotation on a default cluster role or rolebinding to `false`.
+Be aware that missing default permissions and subjects can result in non-functional clusters.
+
+Auto-reconciliation is enabled in Kubernetes version 1.6+ when the RBAC authorizer is active.
+-->
+### 自动更新
+
+在每次启动时，API Server 都会更新默认 ClusterRole 所缺少的各种权限，并更新默认 ClusterRoleBinding 所缺少的各个角色（Role）绑定主体。
+这种自动更新机制允许集群去修复一些特殊的修改。
+由于权限和角色（Role）绑定主体在新的 Kubernetes 版本中可能发生变化，所以这样的话也能够保证角色和角色绑定始终保持是最新的。
+
+如果要禁止此功能,请将默认ClusterRole以及ClusterRoleBinding的`rbac.authorization.kubernetes.io/autoupdate`设置成`false`。
+
+注意，缺乏默认权限和角色（Role）绑定主体可能会导致非功能性集群问题。
+
+自动更新功能在 Kubernetes 版本1.6+ 的 RBAC 认证是默认开启的。
+
+<!--
+### Discovery Roles
+
+Default role bindings authorize unauthenticated and authenticated users to read API information that is deemed safe to be publicly accessible (including CustomResourceDefinitions). To disable anonymous unauthenticated access add `--anonymous-auth=false` to the API server configuration.
+
+To view the configuration of these roles via `kubectl` run:
+
+```
+kubectl get clusterroles system:discovery -o yaml
+```
+
+NOTE: editing the role is not recommended as changes will be overwritten on API server restart via auto-reconciliation (see above).
+
+<table>
+<colgroup><col width="25%"><col width="25%"><col></colgroup>
+<tr>
+<th>Default ClusterRole</th>
+<th>Default ClusterRoleBinding</th>
+<th>Description</th>
+</tr>
+<tr>
+<td><b>system:basic-user</b></td>
+<td><b>system:authenticated</b> group</td>
+<td>Allows a user read-only access to basic information about themselves. Prior to 1.14, this role was also bound to `system:unauthenticated` by default.</td>
+</tr>
+<tr>
+<td><b>system:discovery</b></td>
+<td><b>system:authenticated</b> group</td>
+<td>Allows read-only access to API discovery endpoints needed to discover and negotiate an API level. Prior to 1.14, this role was also bound to `system:unauthenticated` by default.</td>
+</tr>
+<tr>
+<td><b>system:public-info-viewer</b></td>
+<td><b>system:authenticated</b> and <b>system:unauthenticated</b> groups</td>
+<td>Allows read-only access to non-sensitive information about the cluster. Introduced in 1.14.</td>
+</tr>
+</table>
+-->
+### Discovery Roles
+
+无论是经过身份验证的还是未经过身份验证的用户，默认角色（Role）的用户读取API被认为是安全的，可以公开访问（包括CustomResourceDefinitions），
+如果要禁用匿名未经过身份验证的用户访问，请在 API server 中添加 `--anonymous-auth=false` 的配置选项。
+
+通过运行命令 `kubectl` 可以查看这些角色（Role）的配置信息:
+
+```
+kubectl get clusterroles system:discovery -o yaml
+```
+
+注意：不建议编辑这个角色（Role），因为更改将在 API server 重启时自动更新时覆盖（见上文）
+
+<table>
+<colgroup><col width="25%"><col width="25%"><col></colgroup>
```

Comment:
please translating the en in table.
and please see carefully requirement
https://github.com/k8smeetup/k8s-official-translation again.